src/main/java/org/traccar/config/Keys.java

```diff
@@ -1755,6 +1755,15 @@ private Keys() {
             List.of(KeyType.CONFIG),
             "time,position,speed,course,accuracy,result");
 
+    /**
+     * Broadcast method. Available options are "multicast" and "redis". By default (if the value is not
+     * specified or does not matches available options) server disables broadcast.
+     */
+    public static final ConfigKey<String> BROADCAST_TYPE = new StringConfigKey(
+            "broadcast.type",
+            List.of(KeyType.CONFIG),
+            "");
```

Comment:
Can we remove this empty string?

Reply:
It prevents NullPointerException to be thrown by broadcast provider injector 😅

Reply:
Yeah, but I would rather handle null separately.